okta/cache/goCache.go

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 - Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cache
+
+import (
+	"net/http"
+	"time"
+
+	patrickmnGoCache "github.com/patrickmn/go-cache"
+)
+
+type GoCache struct {
+	Ttl         time.Duration
+	Tti         time.Duration
+	RootLibrary *patrickmnGoCache.Cache
```

Comment:
since this isn't used outside of this package, I think at the very least `RootLibrary` should be private, `rootLibrary`, and I'm not sure Ttl/Tti should be public either?